### PR TITLE
autodoc: Add :no-value: option to autoattribute and autodata to suppress the default value of the variable

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Feature

### Purpose
- autodoc: Add `:no-value:` option to `autoattribute` and `autodata` directive to suppress the default value of the variable
- This is a refinement version of #8222 
- refs: #8209

### CODE REVIEW

**Comment by ktbarrett** [comment]
Looks good to me. Thanks for this @tk0miya!

**Comment by tk0miya** [comment]
@ktbarrett Thank you for comment. Now I'm waiting for comment from garmin-mjames for a while.

File: tests/test_ext_autodoc_autoattribute.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+@pytest.mark.sphinx('html', testroot='ext-autodoc')
+def test_autoattribute_novalue(app):
+    options = {'no-value': True}
+    actual = do_autodoc(app, 'attribute', 'target.Class.attr', options)
+    assert list(actual) == [
+        '',
+        '.. py:attribute:: Class.attr',
+        '   :module: target',
+        "   :value: 'bar'",
```
Comment (marlonjames) [inline tests/test_ext_autodoc_autoattribute.py]: Should the value be here?

**Review by marlonjames — COMMENTED** [review]
Looks fine to me, thanks for picking this up, I haven't had time recently.
When I look at the Travis build I don't see the new tests.

File: tests/test_ext_autodoc_autoattribute.py (hunk 1 above)
Comment (tk0miya) [reply tests/test_ext_autodoc_autoattribute.py]: Oops. just fixed.

**Comment by tk0miya** [comment]
>When I look at the Travis build I don't see the new tests.

Now we've migrated to GitHub Actions. Travis CI is no longer used.

**Comment by tk0miya** [comment]
Merged. Thank you for your all!

**Pull request merged**